includes/class-disable-blog-admin.php

```diff
@@ -328,7 +335,23 @@ public function remove_menu_pages() {
 			remove_submenu_page( $page, $subpage );
 		}
 
-	}
+    }
+    
+    /**
+     * Filter the body classes for admin screens to toggle on plugin specific styles.
+     *
+     * @param array $classes
+     * 
+     * @return array
+     */
+    function admin_body_class( $classes ) {
+
+        if( $this->has_front_page() )
+            $classes .= ' disabled-blog';
```

Comment:
Is it `$classes[] = ' disabled-blog';` ?

Reply:
The [`admin_body_class`](https://developer.wordpress.org/reference/hooks/admin_body_class/) actually passes a string, whereas `post_class` and `body_class` pass an array. The `$classes .= ' disabled-blog';` is correct, but the doc block on that function was incorrectly calling it out as an array.